### DISCUSSION TITLE
Is now.sh domain going to be deprecated?

### DISCUSSION THREAD

**Discussion by nooblyf** [vercel/vercel#4646] [General] [answered]
I just deployed my app today and the default domain was vercel.app
I was able to add now.sh manually but i was wondering if there are any plans to remove now.sh in the future? because I like now.sh it's short and easy please don't remove it.

**Comment by paulogdm** [comment, marked as answer]
**now.sh** domains will not be removed, but they will no longer be the default. Please refer to https://www.notion.so/vercel/Change-Timeline-fab7622780694753a92c0a396cea7eb6.

Remember, `vercel.app` and `now.sh` are both public suffixes, and you don't own those domains. So I highly recommend that you use a custom domain for any content you want to rely on in the future. 

EDIT: We are redirecting `now.sh` domains to `vercel.app`. You can read more about this change [here](https://vercel.com/changelog/urls-are-becoming-consistent)

**Reply by ctrlplusb** [reply]
Is there any way I can update an existing now.sh deployment, or put a redirect on it to my new domain?

**Reply by ctrlplusb** [reply]
I have a lot of content not controlled by myself pointing to https://easy-peasy.now.sh 

It's for a fairly popular open source library I maintain.

I have since purchased https://easy-peasy.dev via Vercel CLI.

I'd like to do a 301 permanent redirect from https://easy-peasy.now.sh to https://easy-peasy.dev

Is there any way this can be done?

**Reply by styfle** [reply]
Yes, add both domains to your project settings. Then you can select the redirect option on the domain you wish to redirect to the production domain. 

You can even choose the status code https://vercel.com/changelog/domains-can-now-be-redirected-with-a-custom-status-code

**Reply by ctrlplusb** [reply]
Almost 😅  

<img width="788" alt="Screenshot 2021-02-27 at 21 38 02" src="https://user-images.githubusercontent.com/12164768/109401035-2a56ca00-7944-11eb-9e85-9011d4600eb1.png">

Are you aware of any alternatives @styfle?

**Comment by fallaciousreasoning** [comment]
Hey, one of my apps deployed to a now.sh domain  just started redirecting to a vercel.app domain without any warning. Did I miss an announcement?

**Reply by styfle** [reply]
There was an announcement in January https://vercel.com/changelog/urls-are-becoming-consistent
